test/parallel/test-child-process-exec-detached.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+const cp = require('child_process');
+const childPath = path.join(common.fixturesDir, '/parent-process-exec-nonpersistent.js');
+
+let persistentPid = -1;
+let execParentCloses = false;
```

Comment:
This variable isn't necessary. The `common.mustCall()` on the `close` event will handle it.